src/axom/quest/DistributedClosestPoint.hpp

```diff
@@ -113,6 +120,34 @@ axom::ArrayView<primal::Point<double, 3>> ArrayView_from_Node(conduit::Node& nod
   return axom::ArrayView<PointType>(ptr, sz);
 }
 
+/**
+ * \brief Put BoundingBox into a Conduit Node.
+ */
+template <int NDIMS>
+void put_to_conduit_node(const primal::BoundingBox<double, NDIMS>& bb,
+                         conduit::Node& node)
```

Comment:
Please consider renaming this function to something like `put_bounding_box_to_conduit_node` or maybe `bounding_box_to_conduit_node` (just a style suggestion, especially if you rename `get_from_conduit_node`).

I think a function taking a `BoundingBox &` and a `Node &` is appropriate, since we like to work with `Node &` instead of returning them by value.

Reply:
Will do!